src/core/mavlink_mission_transfer.cpp

```diff
@@ -53,6 +53,32 @@ MAVLinkMissionTransfer::download_items_async(uint8_t type, ResultAndItemsCallbac
     return std::weak_ptr<WorkItem>(ptr);
 }
 
+std::weak_ptr<MAVLinkMissionTransfer::WorkItem>
```

Comment:
I'm afraid all MAVLinkMissionTransfer methods will need unit tests in mavlink_mission_transfer_test.cpp.